Calendars.nuspec

```diff
@@ -2,26 +2,30 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    <metadata minClientVersion="2.8.1">
        <id>CClarke.Plugin.Calendars</id>
-       <version>1.1.0</version>
+       <version>1.2.0-preview11</version>
```

Comment:
Yeah, you would think it's that easy, but I think AppVeyor will overwrite this... I'll have to look into that again if going this route

(but why the 11?)